### ISSUE TITLE
geany-plugins sourceforge site should be maintained or removed

### ISSUE THREAD

**Issue by andy5995** [geany/geany-plugins#1049]
Came across this today in a Google search. I didn't realized I was on an outdated site until after I navigated to the plugins site from geany.org.

http://geany-plugins.sourceforge.net/downloads.html

Seems that this SF site should be maintained or removed, or a 301 redirect set up.

**Comment by elextr** [comment]
Agree

Needs "somebody" whose sourceforge account still works to do it</rant>

@eht16 maybe?

**Comment by eht16** [comment]
- I added some badge to the top of the SF project page with a link to https://plugins.geany.org.
- Also disabled the trackers by removing permissions to create tickets and posts.
- Updated the default support URL to point to the issue tracker at SF
- http://geany-plugins.sourceforge.net still shows the old content, I wasn't able to access the file storage but asked SF for their help

@frlan @elextr @b4n @codebrainz if there are no objections, I would try to the delete the whole Geany-Plugins SF project page in a few months. I assume anything pointing to this site is also already pretty outdated, nobody of us keeps maintaining it and so we can also drop it.

Most of this is probably true for the Geany SF project page (except for maintaing release notes).

**Comment by elextr** [comment]
Unless somebody can provide a good reason to maintain a second site then I would say its a waste of any effort maintaining it, and to delete it.

**Comment by codebrainz** [comment]
@eht16 Maybe before deleting we could [snarf it up in an archiver](https://archive.vn/QPeDh) for posterity, including all the other SF pages, and link them from somewhere on the main site?

**Comment by eht16** [comment]
The files http://geany-plugins.sourceforge.net/ were just there generated files from the READMEs in GIT and the non-generated part of the website is in https://github.com/geany/plugins.geany.org versioned.
So, after my SSH pubkey worked today on SF, I deleted the old web content and put a redirect to the current website (without a backup of auto-generated files).

But I created an export of the SF project settings and tracker items and uploaded this backup to geany.org in the home of the user "geany".
